src/components/loader/loader.stories.ts

```diff
@@ -31,7 +31,7 @@ export const inline_NoTest = (): string => html`
 </div>
 `;
 
-export const customTheme_NoTest = (): string => html`
+export const customThemeAndRTL_NoTest = (): string => html`
```

Comment:
This story seems unrelated to `dir` (custom styling + RTL). 

I notice that the simple test is skipped for tests. @eriklharper @benelan can you confirm if this is intentional? If so, is it because the story's animation causes instability? 

If this is not being tested, I think we could skip testing it in RTL for the same reasons. At least until we switch to Chromatic.

Reply:
We shouldn't be taking any snapshots of Loader because every time Screener runs, the Loader position changes due to the animation. In Screener's UI we ignored the component, so every time there were visual regressions it wouldn't show up as a change. But it still took the snapshot every time, which was a waste.

Chromatic may have a better way of handling animated components, I'll look into that once we migrate.

Reply:
That's a great recap, thanks for the info. I'll remove the RTL test and check back with you when we bring in Chromatic to address it then.